studio/src/editor-panel.js

```diff
@@ -183,20 +183,27 @@ export default class EditorPanel extends LitElement {
     getFragmentPartsToUse() {
         let fragmentParts = '';
         let title = '';
-        const surface = Store.search.value.path.toUpperCase();
+        const surface = Store.search.value.path?.toUpperCase();
```

Comment:
I propose to add a getter `Store.surface` with the above expression.

Reply:
Store isn't a class but just an object. I am not sure if setting a getter to Store is a good idea.